### PR TITLE
Add HTML IDs to cluster progress status info

### PR DESCRIPTION
**Pull request by mareklibra**
With this patch the cluster status HTML element on the cluster detail page looks like:
```
<dd id="cluster-progress-status-value" data-pf-content="true" class="">Installing</dd>
````

### CODE REVIEW

**Comment by mareklibra** [comment]
cc @ukalifon

File: src/components/clusterDetail/ClusterProgress.tsx [hunk 1]
```diff
@@ -67,8 +67,16 @@ const ClusterProgress: React.FC<ClusterProgressProps> = ({ cluster }) => {
   return (
     <>
       <DetailList>
-        <DetailItem title="Started on" value={getHumanizedDateTime(cluster.installStartedAt)} />
-        <DetailItem title="Status" value={getInstallationStatus(cluster)} />
+        <DetailItem
+          title="Started on"
+          value={getHumanizedDateTime(cluster.installStartedAt)}
+          idPrefix="cluster-progress-started-on"
```
Comment (jtomasek) [inline src/components/clusterDetail/ClusterProgress.tsx:73]: Nit: we could simply call it `id`

Comment (mareklibra) [reply src/components/clusterDetail/ClusterProgress.tsx:73]: But we are further appending it so I thought this is clear that such ID will not appear in the DOM

**Pull request merged**